setup.cake

```diff
@@ -155,7 +156,7 @@ BuildParameters.Tasks.DotNetCoreBuildTask
 
             // If .NET Core project, copy using dotnet publish for each target framework
             // Otherwise just copy
-            if(parsedProject.IsNetCore)
+            if(parsedProject.IsVS2017ProjectFormat)

```

Comment:
@mholo65 why did you make this change?  What was the requirement?  Could this be the same issue that @devlead is facing when using Cake.Recipe in a few projects that he has?

Reply:
I noticed that after `Cake.Recipe`was updated (and indirectly `Cake.Incubator`) `dotnet publish` wasn't run anymore for `Cake.Bakery` due to the fact that it's not a NETCore project. I.e. I only target `net461`... Did a corresponding PR to Cake.Recipe, here https://github.com/cake-contrib/Cake.Recipe/pull/205

Reply:
But due to the face that we don't have a solution for https://github.com/cake-contrib/Cake.Recipe/pull/176 yet, I need to use my own copy of `DotNetCore-Build` in order to build under Mono,